machine-learning/tutorials/TransferLearningTF/Program.cs

```diff
@@ -200,6 +152,37 @@ public static IEnumerable<ImageData> ReadFromTsv(string file, string folder)
              });
             // </SnippetReadFromTsv>
         }
-    }
 
+        // <SnippetInceptionSettings>
+        private struct InceptionSettings
+        {
+            public const int ImageHeight = 224;
+            public const int ImageWidth = 224;
+            public const float Mean = 117;
+            public const float Scale = 1;
+            public const bool ChannelsLast = true;
+        }
+        // </SnippetInceptionSettings>
+
+
+        // <SnippetDeclareImageData>
+        public class ImageData
+        {
+            [LoadColumn(0)]
+            public string ImagePath;
+
+            [LoadColumn(1)]
+            public string Label;
+        }
+        // </SnippetDeclareImageData>
+
+        // <SnippetDeclareImagePrediction>
+        public class ImagePrediction : ImageData
```

Comment:
This is inheriting and works in this case because the way the pipeline is structured. Since we removed this practice from the How-To's do we still want to keep it here?

Reply:
As discussed, we'll leave this here for now, as revise later if necessary